### PR TITLE
[IMPROVE] Layout of livechat manager pages to new style

### PR DESCRIPTION
**Pull request by ggazzo**
image
image
image
image
image

### CODE REVIEW

File: app/livechat/client/views/app/livechatCurrentChats.html [hunk 1]
```diff
@@ -59,11 +97,41 @@
 							{{/requiresPermission}}
 						</tr>
 					{{/each}}
+					{{#unless isReady}}
+						<tr class="table-no-click">
+							<td class="table-loading-td">{{> loading}}</td>
```
Comment (sampaiodiego) [inline app/livechat/client/views/app/livechatCurrentChats.html]:
```suggestion
							<td class="table-loading-td" colspan="6">{{> loading}}</td>
```

**Comment by engelgabriel** [comment]
@ggazzo can we move all the "Add New" buttons to the top?

**Comment by ggazzo** [comment]
> @ggazzo can we move all the "Add New" buttons to the top?

I could, but it's more expensive... today the header its autogenerated :/

**Comment by renatobecker** [comment]
@ggazzo, after clicking 2X on the `Load more` button(Current Chats), the button disappears.
Did you notice that?

**Comment by ggazzo** [comment]
> @ggazzo, after clicking 2X on the `Load more` button(Current Chats), the button disappears.
> Did you notice that?
> 
> Screen Shot 2019-05-29 at 21 06 47
If I'm not mistaken, the `load more` only appears IF we have something more to load... we could disable instead of hide... but its not a bug I think

**Pull request merged**